Deployment/Mirantis-Guidance.md

```diff
@@ -0,0 +1,274 @@
+
+After successful migration, the unused Node Type should be removed.
+- [How to remove a Service Fabric node type](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-how-to-remove-node-type)
+- [Remove the original node type and cleanup its resources](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-scale-up-primary-node-type#remove-the-original-node-type-and-cleanup-its-resources)
+
+Additional guidance related to container runtimes
+- Modifications to the Docker data root (C:\ProgramData\docker) must be tested before doing the migration.
+- Other container runtimes like for example containerd or DockerEE should not be installed side-by-side with Mirantis Container Runtime.
+
+Additional information about the Mirantis OS image in Azure Marketplace
+
+This image will have MCR preinstalled and will receive regular updates for container runtime and Windows. Mirantis will make this VM image publicly available in early September 2022 for customers to switch any workloads to. The cost of this marketplace image will include the cost of support and license for the runtime.
+Service Fabric customers must follow the official scale-up process to create a new node type and move over the container workloads.
+
+## Scenario 3: Customer has Azure Service Fabric Managed Cluster using deprecated Windows Server images with-Containers, and does NOT host containers in Docker as part of their applications
+
+Customers in this scenario can simply use the in-place upgrade to switch to another OS SKU without container support and moving the workloads.
+Documentation:
+- [Modify the OS SKU for a node type](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-cluster-modify-node-type#modify-the-os-sku-for-a-node-type)
+- [Configuring placement constraints for Service Fabric services](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-resource-manager-configure-services#placement-constraints)
+- [Service Fabric application upgrade](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-upgrade)
+
+Scenario 4: Customer has Azure Service Fabric Managed Cluster using deprecated Windows Server images with-Containers, and does host containers in Docker as part of their applications
+
+Service Fabric Managed Cluster release support for custom OS images in beginning of September 2022. Please stay tuned for the official announcement.
+Documentation:
+- [Get started: Prep Windows for containers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment)
+- [Automatic OS image upgrade for custom images](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade#automatic-os-image-upgrade-for-custom-images)
+
+
+## Frequently Asked Questions
+
+For all further questions please reach out to your account team or create a Microsoft support case.
```

Comment:
[create a Microsoft support case](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request)

Reply:
Done